.github/workflows/test.yml

```diff
@@ -187,6 +187,23 @@ jobs:
           name: "${{ github.sha }}-${{ matrix.part }}-race-output"
           path: ./${{ matrix.part }}-race-output.txt
 
+  test-rosetta:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+      - uses: technote-space/get-diff-action@v4
+        id: git_diff
+        with:
+          PATTERNS: |
+            **/**.go
+            go.mod
+            go.sum
+      - name: test rosetta
+        run: |
+          make test-rosetta
+        # if: env.GIT_DIFF
```

Comment:
Shouldn't this be enabled? @marbar3778

Reply:
it should be but was disabled so I could run the CI in the branch.